### ISSUE TITLE
NewlineAtEndOfFile does not work for files other than Java

### ISSUE THREAD

**Issue by mkordas** [teamed/qulice#522]
Good practice is to end all source and text files with newline. A text file under Unix, consists of a series of lines, each of which ending with a newline character (`\n`). It is required by POSIX standard:
* http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Currently Qulice checks [only Java files](https://github.com/teamed/qulice/blob/master/qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml#L50) according to this rule:
```xml
    <module name="NewlineAtEndOfFile">
        <property name="fileExtensions" value="java"/>
        <property name="lineSeparator" value="lf"/>
    </module>
```

This should be extended to handle all typical extensions that are known to be text-based files in typical Maven projects.

**Comment by krzyk** [comment]
@mkordas could you join this issue with #521? If there are too many obvious *one line fix bugs* we will burn to much budget in a project.

**Comment by mkordas** [comment]
@krzyk merged with #521
